pkg/agent/openflow/client.go

```diff
@@ -432,6 +432,8 @@ func (c *client) InstallNodeFlows(hostname string,
 		} else {
 			// Since broadcast is not supported in IPv6, ARP should happen only with IPv4 address, and ARP responder flows
 			// only work for IPv4 addresses.
+			// arpResponderFlow() uses globalVirtualMAC for gateway IP as a virtual destination MAC address.
```

Comment:
Probably change to: arpResponderFlow() adds a flow to resolve peer gateway IPs to globalVirtualMAC.

Reply:
Updated.